src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpMetadataContext.cs

```diff
@@ -1,26 +1,35 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Collections.Immutable;
+using System.Linq;
 using Microsoft.CodeAnalysis.ExpressionEvaluator;
 
 namespace Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator
 {
-    internal sealed class CSharpMetadataContext : MetadataContext
```

Comment:
Is this change required for portability (i.e. is this just changing to make it a struct instead of a class)?